contracts/extensions/ReputationBootstrapper.sol

```diff
@@ -0,0 +1,191 @@
+/*
+  This file is part of The Colony Network.
+
+  The Colony Network is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  The Colony Network is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with The Colony Network. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+pragma solidity 0.7.3;
+pragma experimental ABIEncoderV2;
+
+import "./../../lib/dappsys/erc20.sol";
+import "./../reputationMiningCycle/IReputationMiningCycle.sol";
+import "./../colonyNetwork/IColonyNetwork.sol";
+import "./ColonyExtensionMeta.sol";
+
+// ignore-file-swc-108
+
+
+contract ReputationBootstrapper is ColonyExtensionMeta {
+
+  // Constants
+
+  uint256 constant INT128_MAX = 2**127 - 1;
+  uint256 constant SECURITY_DELAY = 1 hours;
+
+  // Events
+
+  event GrantSet(bool paid, bytes32 hashedSecret, uint256 reputationAmount);
+  event GrantClaimed(address recipient, uint256 reputationAmount, bool paid);
+
+  // Data structures
+
+  struct Grant {
+    uint256 amount;
+    uint256 timestamp;
+  }
+
+  // Storage
+
+  address public token;
```

Comment:
Is there a reason we've gone with public variables rather than our own defined getters? I like it, but just wondering.

Reply:
Good question. I was going to say "because it's not an EtherRouter" but then I would be thinking of the CoinMachine contract, in which we made them public for that reason. Perhaps because there's no interface contract? I can't fully remember, and going over other extensions I see that it isn't the style we use elsewhere.

Reply:
In the spirit of consistency I'll make them private and add getters.